File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/SpotifyTopTracks.java
package com.joxad.zikobot.data.module.spotify_api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.Data;

/**
 * Created by Jocelyn on 25/02/2017.
 */
@Data
public class SpotifyTopTracks {
    @SerializedName("tracks")
    private List<SpotifyTrack> tracks;
}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/SpotifyTopTracks.kt
package com.joxad.zikobot.data.module.spotify_api.model

import com.google.gson.annotations.SerializedName

/**
 * Created by Jocelyn on 25/02/2017.
 */
class SpotifyTopTracks {
    @SerializedName("tracks")
    val tracks: List<SpotifyTrack>? = null
}
